Makefile

```diff
@@ -59,4 +63,9 @@ vagrant-sync-build: build
 vagrant-deploy: vagrant-sync-config vagrant-sync-build
 	export KUBECTL="cluster/kubectl.sh --core" && ./cluster/deploy.sh
 
+.glide.status: glide.yaml
+	if test -f $@; then \
```

Comment:
Upon first switching to this approach (or after using distclean), all developer's systems will have a glide.yaml but not a .glide.status. It makes more sense to assume an "install" is needed in that case--as opposed to running update--which would get everybody's glide.lock out of sync